### DISCUSSION TITLE
About the .MSI for v1.9.12

### DISCUSSION THREAD

**Discussion by Lavdaft** [michaelrsweet/htmldoc#448] [General]
Hello Michael,

I have one question: the v1.9.12 has no .msi file to download and install. Should we compile by ourselves?

**Comment by michaelrsweet** [comment]
@Lavdaft For now, yes. I was having trouble getting the NuGet packages to work with the Visual Studio project. Hoping to get back to doing the MSI soon, I've just been swamped with other work.

**Reply by Lavdaft** [reply]
Thank you so much for your reply :)

**Reply by Lavdaft** [reply]
@michaelrsweet, can you please provide the executables? I am having some issues with the compilation. I can install it manually, but it would be nice to have the executables. Since the version 12 is realeased, I believe you're able to compile right?

**Reply by michaelrsweet** [reply]
@Lavdaft The issue is linking and packaging the resulting content - as soon as you update a dependent package you have to update raw paths within both the Visual Studio and Advance Installer projects for each DLL, plus a helper script to generate the documentation PDF. I just haven't had time to go through all of that and test it (and since this is free software with *no* support or warranty...)

**Comment by alexgoa** [comment]
Hello, thanks for your work, Michael.
I have the same issue. May I suggest that if somebody has succeeded in complying with the msi, he make the file available for others?
thanks
Ale
